rpcd/playbooks/roles/rpc_maas/tasks/package_install.yml

```diff
@@ -21,9 +21,28 @@
     cache_valid_time: 600
   with_items: maas_apt_packages
 
-- name: Install pip packages
+- name: Create venv dir for maas
+  file:
+    path: "{{ maas_venv }}/"
+    state: directory
+
+- name: Copy pip conf to venv
+  copy: src=/root/.pip/pip.conf dest="{{ maas_venv }}/pip.conf"
+
+- name: Update pip
+  pip:
+    name: pip
+    extra_args: '--upgrade'
+    virtualenv: "{{ maas_venv_enabled | bool | ternary(maas_venv + '/' | dirname, omit) }}"
```

Comment:
@prometheanfire We were having an issue where without the trailing slash the venv would be installed into the directory above mass_venv and not inside the mass_venv directory.